### ISSUE TITLE
Support generating multiple header files

### ISSUE THREAD

**Issue by ibachar** [wxFormBuilder/wxFormBuilder#272]
Hi,

I think it would be good it were possible to generate multiple header files (Maybe tell each top level panel / form its header file) since for large projects, making a change to the gui causes all cpp files to recompile

Thanks.

**Comment by jhasse** [comment]
Good idea. A workaround is to create a separate project for each panel. Would that work for you?

**Comment by ibachar** [comment]
Yes, the workaround will probably work.
